src/components/EditSection.tsx

```diff
@@ -46,40 +53,25 @@ export const EditSection: React.FunctionComponent<EditSectionProps> = ({ page })
                 <span className="text-muted mx-1" aria-hidden="true">
                     •
                 </span>{' '}
-                <Link href={`https://github.com/sourcegraph/handbook/commits/main/${CONTENT_FOLDER}/${page.path}`}>
+                <Link href={`https://github.com/sourcegraph/handbook/commits/main/${CONTENT_FOLDER}/${page.path.replace(/^\//, '')}`}>
                     History
                 </Link>
             </div>
         )}
         <div className="sidebar-bottom-links">
-            {page.path === 'company/team/index.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/team.yml">
-                    Edit this data on GitHub
-                </Link>
-            )}
-            {page.path === 'product/product_areas.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/product_areas.yml">
-                    Edit this data on GitHub
-                </Link>
-            )}
-            {page.path === 'product/feature_maturity.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/features.yml">
-                    Edit this data on GitHub
-                </Link>
-            )}
-            {page.path === 'product/feature_compatibility.md' && (
-                <Link href="https://github.com/sourcegraph/handbook/edit/main/data/features.yml">
-                    Edit this data on GitHub
+            {Object.keys(GENERATED_PAGE_DATA).map(pagePath => {
+                if (page.path === pagePath) {
+                    return <Link href={`https://github.com/sourcegraph/handbook/edit/main/data/${GENERATED_PAGE_DATA[pagePath]}`} >
+                        Edit this page
+                    </Link>
+                }
+            })}
+
+            {!Object.keys(GENERATED_PAGE_DATA).includes(page.path) && (
+                <Link href={`https://github.com/sourcegraph/handbook/edit/main/${CONTENT_FOLDER}/${page.path.replace(/^\//, '')}`}>
```

Comment:
The `.replace(/^\//, '')` is needed to prevent a warning for when the path is `/` and it causes a double `//` in the href value.